.eslintrc.js

```diff
@@ -2,6 +2,7 @@
 module.exports = {
   root: true,
   extends: [require.resolve('./tooling/config-eslint/node')],
+  ignorePatterns: ['apps/**', 'packages/**', 'tooling/**'],
```

Comment:
Correct me if I'm wrong but doesn't this remove linting from the majority of the repo?

Reply:
This file only gets used by the root of the repository. In workspaces, you'll find `.eslintrc.js` files that have `root: true` so they won't resolve any rules higher than there.

I do seem to have a misconfiguration elsewhere, though. 😄

Reply:
Ah, nope, I'm all good. Forgot that we have a lot of rules turned off right now in `tooling/config-eslint/utils/rules.js`.

If you want to see the linter get more strict, you can comment out some rules in there and run `turbo lint --force` and you'll be given issues.